main.go

```diff
@@ -101,13 +117,6 @@ func main() {
 
 					return Lint(ctx.String("file"), ctx.String("schema"))
 				},
-			}, {
-				Name:  "test",
```

Comment:
I hope it's okay if we stay with the command names we decided on internally. We wanted to adopt the dbt syntax and name this command `test`. I know, this might unintuitive for you as Soda users.

One reason behind this, is to keep the commands small.

Reply:
Ok, that is fine. Fixed in [6c0218e](https://github.com/datacontract/cli/pull/3/commits/6c0218eea40c942872fa9fdfc09a8415a1f32d8c)

As you say, I do not find it so intuitive to have a `quality` command just printing the quality check specification, and a `test` command to check the quality, whereas in many software utilities, the test command is usually reserved for testing purpose of the software itself. All is a question of proper documentation, so it is fine :)